clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/common/AzureUtilities.groovy

```diff
@@ -84,18 +113,34 @@ class AzureUtilities {
   }
 
   static String getAppNameFromAzureResourceName(String azureResourceName) {
+    if (azureResourceName == null) {
```

Comment:
Personally, I believe that if we are checking the argument and it's not valid (i.e., null) we should throw an InvalidArgumentException and let the caller handle the exception. Question is, "Is 'null'  an invalid argument for these methods?"